RequestManagementSystem/Client/Request.py

```diff
@@ -534,10 +534,13 @@ def optimize( self ):
     repAndRegList = []
     removeRepList = []
     i = 0
-    while i < len( self.__operations__ ) - 1:
```

Comment:
Ah, yes! I was scared that since the number of operations is changing within the loop the length would not be reevaluated at each loop (kind of optimisation)... I now checked with an example and this is not the case. len() is reevaluated at each loop... Changed back...